src/main/java/com/ikubinfo/service/AttributeService.java

```diff
@@ -0,0 +1,71 @@
+package com.ikubinfo.service;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.ikubinfo.converter.AttributeConverter;
+import com.ikubinfo.dto.AttributeDto;
+import com.ikubinfo.entities.AttributeEntity;
+import com.ikubinfo.enums.AttributeType;
+import com.ikubinfo.repository.AttributeRepository;
+
+@Service
+@Transactional
+public class AttributeService {
+
+	@Autowired
+	private AttributeRepository attributeRepository;
+
+	@Autowired
+	private AttributeConverter attributeConverter;
+
+	public AttributeDto createAttribute(AttributeDto attributeToBeCreated) {
+		if (AttributeType.contains(attributeToBeCreated.getType())) {
+			AttributeEntity attribute = attributeConverter.toEntity(attributeToBeCreated);
+			return attributeConverter.toDto(attributeRepository.save(attribute));
+		} else {
+			return null; // replace with exception
```

Comment:
practice: krijo disa types of exceptions. jane klasa te thjeshta qe do bejne `extends RuntimeException`, psh `ValidationException`, qe do kete disa constructors (shif constructors nga RuntimeException), por qe do perdoret me shume ai me String, qe te kesh nje custom message